documentation/assemblies/oauth/assembly-oauth-authentication.adoc

```diff
@@ -10,7 +10,7 @@ Strimzi supports the use of OAuth 2.0 authentication using the _SASL OAUTHBEARER
 OAuth 2.0 enables standardized token-based authentication and authorization between applications, using a central authorization server to issue tokens that grant limited access to resources.
 
 You can configure OAuth 2.0 authentication, then xref:assembly-oauth-authorization_{context}[OAuth 2.0 authorization].
-OAuth 2.0 authentication can also be used in conjunction with xref:simple-acl-{context}[ACL-based Kafka authorization].
+OAuth 2.0 authentication can also be used in conjunction with xref:type-AclRule-reference[ACL-based Kafka authorization].
```

Comment:
I think this should link to some chapter about configuring authorization in Kafka CR. 

Open Policy Agent can be used with OAuth 2 authentication as well. So we should mention it here as well.

Reply:
> OAuth 2.0 authentication can also be used in conjunction with ACL or OPA-based xref:on-securing-kafka-authorization-str[Kafka authorization].